tffm/models.py

```diff
@@ -25,14 +18,39 @@ class TFFMClassifier(TFFMBaseModel):
     """
 
     def __init__(self, **init_params):
-        assert 'loss_function' not in init_params
+        assert 'loss_function' not in init_params, "Parameter 'loss_function' is unexpected for TFFMClassifier"
```

Comment:
I thought the whole point was to allow for arbitrary custom loss functions for a `TFFMClassifier`. Why are we prohibiting this?